tests/openwisp2/settings.py

```diff
@@ -16,6 +17,15 @@
     }
 }
 
+TIMESERIES_DATABASE = {
+    'BACKEND': 'openwisp_monitoring.db.backends.influxdb',
+    'USER': 'openwisp',
+    'PASSWORD': 'openwisp',
+    'NAME': 'openwisp2',
+    'HOST': 'localhost',
+    'PORT': '8086',
```

Comment:
for backward compatibility, please make sure the application works also if this is not defined, supply the default values somewhere, but only if they're not defined (and log a warning using the logger, urging to migrate).
We'll then remove this once all instances are upgraded.